### ISSUE TITLE
Not really building x86 binaries?

### ISSUE THREAD

**Issue by dabrahams** [c3lang/win-llvm#2]
From looking at the code, I think the things being labeled as x86 binaries are actually x64 binaries; `TARGET_CPU` doesn't get used for anything but choosing the names of files.

**Comment by lerno** [comment]
Yes, I believe I just used the original script and just purged some unnecessary stuff and fixed what needed fixing until it worked. I didn't need x86_32 so I didn't fix that.

**Comment by lerno** [comment]
Do you need it to work?

**Comment by dabrahams** [comment]
No thanks; I have [my own setup](https://github.com/hylo-lang/llvm-build) now, but I thought you should know as many people probably, like me, fork this project thinking they're getting something else.

**Comment by CodeOfDestruction** [comment]
it would be cool if you could release x86 binaries too ? Thanks
